[2025-02-18T06:55:16.980+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T06:55:17.012+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.email_notification manual__2025-02-18T06:54:55.611963+00:00 [queued]>
[2025-02-18T06:55:17.031+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.email_notification manual__2025-02-18T06:54:55.611963+00:00 [queued]>
[2025-02-18T06:55:17.033+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-18T06:55:17.057+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): email_notification> on 2025-02-18 06:54:55.611963+00:00
[2025-02-18T06:55:17.066+0000] {standard_task_runner.py:72} INFO - Started process 4294 to run task
[2025-02-18T06:55:17.075+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'email_notification', 'manual__2025-02-18T06:54:55.611963+00:00', '--job-id', '171', '--raw', '--subdir', 'DAGS_FOLDER/DAG-fraud.py', '--cfg-path', '/tmp/tmphem4fzco']
[2025-02-18T06:55:17.078+0000] {standard_task_runner.py:105} INFO - Job 171: Subtask email_notification
[2025-02-18T06:55:17.543+0000] {task_command.py:467} INFO - Running <TaskInstance: ml_pipeline.email_notification manual__2025-02-18T06:54:55.611963+00:00 [running]> on host 8538735272b1
[2025-02-18T06:55:18.175+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='group1' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='email_notification' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T06:54:55.611963+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-18T06:54:55.611963+00:00'
[2025-02-18T06:55:18.183+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T06:55:18.765+0000] {baseoperator.py:423} WARNING - EmailOperator.execute cannot be called outside TaskInstance!
[2025-02-18T06:55:18.894+0000] {base.py:84} INFO - Retrieving connection 'smtp_default'
[2025-02-18T06:55:18.896+0000] {email.py:272} INFO - Email alerting: attempt 1
[2025-02-18T06:55:20.291+0000] {email.py:283} INFO - Sent an alert email to ['realjokiin@gmail.com']
[2025-02-18T06:55:21.930+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T06:55:21.951+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T06:55:21.955+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=email_notification, run_id=manual__2025-02-18T06:54:55.611963+00:00, execution_date=20250218T065455, start_date=20250218T065517, end_date=20250218T065521
[2025-02-18T06:55:22.048+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T06:55:22.077+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-02-18T06:55:22.107+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T06:55:22.110+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
